cluster-provision/tools/Makefile

```diff
@@ -0,0 +1,29 @@
+SHELL := /bin/bash
+
+BIN_DIR = $(CURDIR)/build
+GO ?= go
+
+export GO111MODULE=on
+export GOPROXY=direct
+export GOSUMDB=off
+export GOFLAGS=-mod=vendor
+
+
+all: push
+
+.PHONY: fmt
+fmt:
+	$(GO) fmt ./check-image-pull-policies/...
+
+.PHONY: container
+container: fmt
+	docker build -f check-image-pull-policies/Dockerfile -t kubevirtci/check-image-pull-policies .
+
+.PHONY: vendor
+vendor:
+	$(GO) mod tidy
+	$(GO) mod vendor
+
+.PHONY: push
+push: container
```

Comment:
Pushing this is done manually ? or do we need some prow infra like gocli ? 

Also  we may have in the future a tarball with scripts and binaries at kubevirtci release page that tarball can contain this binary too, https://github.com/kubevirt/kubevirtci/pull/403

Reply:
I've pushed it manually for now, was sufficient. Not sure whether we do need updates on a regular basis.

Reply:
Ack, let's be reactive if we see a lot of updates we put some infra.